libraries/libfc/src/crypto/city.cpp

```diff
@@ -38,8 +38,10 @@
 
 #if defined(__SSE4_2__) && defined(__x86_64__)
 #include <nmmintrin.h>
+#define MM_CRC32_U64I _mm_crc32_u64
 #else
-uint64_t _mm_crc32_u64(uint64_t a, uint64_t b );
+uint64_t mm_crc32_u64(uint64_t a, uint64_t b );
+#define MM_CRC32_U64I mm_crc32_u64
```

Comment:
Should we use `FC_MM_CRC32_U64I` instead?

Reply:
I was trying to keep the same length to avoid touching a bunch of whitespace over in `CHUNK()` :sweat_smile: 

Since this define is local to a .cpp it doesn't seem like there is much risk of a clash? Possibly `mm_crc32_u64()` should be moved in to fc's namespace though?

Reply:
actually if `_mm_crc32_u64` is just placed in to a namespace I think that's safe. maybe that's the better choice :thinking:

Reply:
I didn't notice this was only in this one source file. I think it is fine as is.